pkg/api/v20170701/const.go

```diff
@@ -0,0 +1,77 @@
+package v20170701
+
+const (
+	// APIVersion is the version of this API
+	APIVersion = "2017-07-01"
+)
+
+// the orchestrators supported by 2017-07-01
+const (
```

Comment:
The API model still has to support 'Swarm' orchestrator for scale up scenarios and also because we haven't deprecated this orchestrator. Without Swarm as a const all scale up calls will fail because the RP won't be able to deserialize the orchestrator into OrchestratorType const.